src/Symfony/Bundle/FrameworkBundle/Test/WebTestAssertions.php

```diff
@@ -0,0 +1,382 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Test;
+
+use PHPUnit\Framework\Assert;
+use Symfony\Bundle\FrameworkBundle\Client;
+use Symfony\Component\DomCrawler\Crawler;
+use Symfony\Component\HttpFoundation\Cookie;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Ideas borrowed from Laravel Dusk's assertions.
+ *
+ * @see https://laravel.com/docs/5.7/dusk#available-assertions
+ */
+trait WebTestAssertions
+{
+    /** @var Client|null */
+    protected static $client;
+
+    public static function assertResponseIsSuccessful(): void
+    {
+        $response = static::getResponse();
+
+        Assert::assertTrue(
+            $response->isSuccessful(),
+            "Response was expected to be successful, but actual HTTP code is {$response->getStatusCode()}."
+        );
+    }
+
+    public static function assertHttpCodeEquals(int $expectedCode): void
+    {
+        Assert::assertSame(
+            $expectedCode,
+            $code = static::getResponse()->getStatusCode(),
+            "Response code [$expectedCode] does not match actual HTTP code [$code]."
+        );
+    }
+
+    public static function assertResponseHasHeader(string $headerName): void
+    {
+        Assert::assertTrue(
+            static::getResponse()->headers->has($headerName),
+            "Header [$headerName] was not found in the Response."
+        );
+    }
+
+    public static function assertResponseNotHasHeader(string $headerName): void
+    {
+        Assert::assertFalse(
+            static::getResponse()->headers->has($headerName),
+            "Header [$headerName] was not expected to be found in the Response."
+        );
+    }
+
+    public static function assertResponseHeaderEquals(string $headerName, string $expectedValue): void
```

Comment:
`array $expectedValue`

Reply:
I'll go for `mixed` instead then, it can be different types